src/foam/u2/wizard/StepWizardletController.js

```diff
@@ -114,6 +141,17 @@ foam.CLASS({
         section: this.sections[0][0],
         data$: this.wizardlets[0].data$,
       });
+
+      // If the first screen has no available sections, move next and repeat.
```

Comment:
I'd actually like to see the function f - be named something more relevant and remove the comment on 145

Reply:
I'll add a better function name so that the stack trace looks better if there's ever an error here, but why remove the comment?